content/en/tracing/guide/inferred-service-opt-in.md

```diff
@@ -0,0 +1,264 @@
+---
+title: New Service page and Inferred Services
+kind: Guide
+disable_toc: false
+private: true
+further_reading:
+- link: "/tracing/services/"
+  tag: "Documentation"
+  text: "Service Observability"
+- link: "/tracing/trace_collection/"
+  tag: "Documentation"
+  text: "Sending Traces to Datadog"
+- link: "/tracing/trace_collection/dd_libraries/"
+  tag: "Documentation"
+  text: "Add the Datadog Tracing Library"
+---
+
+{{< callout url="https://docs.google.com/forms/d/1imGm-4SfOPjwAr6fwgMgQe88mp4Y-n_zV0K3DcNW4UA/edit" d_target="#signupModal" btn_hidden="true" btn_hidden="false" header="Opt in to the private beta!" >}}
+Inferred services and the new Service Page layout are in private beta. To request access, complete the form.
+{{< /callout >}}
+
+Follow the steps below to enable the new dependency map on the [Service Page][1], and add _Inferred services_ to the [Service Map][2] and [Service Catalog][3] pages.
+
+## Inferred services
+
+Datadog can automatically discover the dependencies for an instrumented service, such as a database or a third-party API even if that dependency hasn't been instrumented yet. Datadog infers the presence of these dependencies and their performance based on information on the exit spans, outgoing spans, and outbound requests of your instrumented services.
+
+To determine the names and types of the inferred services, Datadog uses span attributes. Inferred external APIs use the default naming scheme `net.peer.name`. For example, `api.stripe.com`, `api.twilio.com`, `us6.api.mailchimp.com`. Inferred databases use the default naming scheme `db.instance`.
+
+If you're using the Go, Java, NodeJS, PHP, or Ruby tracer, you can customize the default names for inferred services. For more information, see the "Peer Service Mapping" section for your language below.
```

Comment:
Optional: Not sure if it would be better to link to the #apm-tracer-configuration section instead since a user has to select their language.
```suggestion
If you're using the Go, Java, NodeJS, PHP, or Ruby tracer, you can customize the default names for inferred services. For more information, see the [Peer Service Mapping](#peer-service-mapping) section for your language below.
```

Reply:
Yeah, so I actually originally did this - but those sections don't exist in all languages, so the link would break if they had Python selected. Another option was to add the section and then just say it wasn't available. But that felt like rug pulling